src/_guides/openapi/specification/v3.1/understanding-structure/basic-structure.md

```diff
@@ -0,0 +1,317 @@
+
+### 7. Webhooks Object
+
+```yaml
+webhooks:
+  newBooking:
+    post:
+      operationId: new-booking
+      summary: New Booking
+      description: |
+        Subscribe to new bookings being created, to update integrations for your users.  Related data is available via the links provided in the request.
+      tags:
+        - Bookings
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Booking'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully.
+```
+
+### 8. Tags Object
+
+You may have spotted the `tags` keyword in the paths and webhooks, and those are referencing tags defined in the top-level `tags` object. The tag name is used to group related operations together. Each tag has a name and an optional description.
+
+```yaml
+tags:
+  - name: Bookings
+    description: | 
+      Create and manage bookings for train trips, including passenger details
+      and optional extras.
+  - name: Payments
+    description: |
+      Pay for bookings using a card or bank account, and view payment
+      status and history.
+
+      > warn
+      > Bookings usually expire within 1 hour so you'll need to make your payment
+      > before the expiry date 
+```
+
+The name is often displayed to users in human-readable documentation so its best to make it "Title Case", and the description can be quite long, think paragraphs not sentences, explaining what this concept is to the user as that will also show up in most documentation tools.
+
+## Example OpenAPI Document
+
+Putting it all together, here is a simple example of an OpenAPI document:
+
+```yaml
+openapi: 3.0.3
+info:
+  title: Sample API
+  description: A sample API to illustrate OpenAPI concepts.
+  version: 1.0.0
+  contact:
+    name: API Support
+    url: http://www.example.com/support
+    email: support@example.com
+servers:
+  - url: https://api.example.com/v1
+    description: Production
+paths:
+  /users:
+    get:
+      summary: List all users
+      responses:
+        '200':
+          description: A list of users
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/User'
+    post:
+      summary: Create a new user
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/User'
+      responses:
+        '201':
+          description: User created
+components:
+  schemas:
+    User:
+      type: object
+      properties:
+        id:
+          type: integer
+          format: int64
+        username:
+          type: string
+        email:
+          type: string
+          format: email
+security:
+  - api_key: []
+tags:
+  - name: users
```

Comment:
The tag doesn't seem to be used in the `paths` operations. Might be good to add it on each operation of this example?